src/index.js

```diff
@@ -24,8 +24,22 @@ const fetchEntities = async ({ endpoint }, ctx) => {
   return entities;
 };
 
+const isDynamicZone = (node) => {
+  // Dynamic zones are always arrays
+  if (isArray(node)) {
+    return node.some((nodeItem) => {
+      if (isPlainObject(nodeItem)) {
```

Comment:
you don't need to check for isPlainObject with has(x, key) it is safe to use on any type